sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/BlobServiceProperties.java

```diff
@@ -32,6 +32,9 @@
     /** @return the deleteRetentionPolicy value. */
     DeleteRetentionPolicy deleteRetentionPolicy();
 
+    /** @return the containerDeleteRetentionPolicy value. */
```

Comment:
nit, it would be better if the javadoc is more meanful (these bad old ones are likely from codegen).

Reply:
Changed, thanks!